libpod/image/image.go

```diff
@@ -923,6 +923,11 @@ func (i *Image) Size(ctx context.Context) (*uint64, error) {
 	return nil, errors.Wrap(err, "unable to determine size")
 }
 
+//VirtualSize returns the virtual size of the image
+func (i *Image) VirtualSize(ctx context.Context) (*uint64, error) {
+	return i.Size(ctx)
```

Comment:
The doubled call to `i.Size()` should be avoided - it's a very expensive operation.

Reply:
@mheon What would you propose:
- to make a new field which is lazy initialized, and if called a second time no calculation is needed (I suppose image size cannot change)
- to remove VirtualSize function, and to always use Size because both attributes always have the same values. At the moment Size == VirtualSize is more or less everywhere I saw in code, but I do not know if this is just temporary solution, or if this will change. I did VirtualSize function just to have an abstraction for calculation, but if both values are always the same, it is not needed.

Or do you have any other ideas?